topics/icrc_28_trusted_origins.md

```diff
@@ -6,107 +6,63 @@
 
-In case the global authentication flow ([ICRC-34](./icrc_34_get_global_delegation.md)) is used, a delegation chain for
-the user's global identity is obtained. Since a global identity is not scoped per relying party, canisters from other
-relying parties can be called on behalf of the user without user approval unless the delegation chain is restricted.
+## Methods
 
-Therefore, a signer **MUST** restrict the delegation chain for the user's global identity to a set of canister
-targets that trust the relying party that requested the delegation chain. This standard describes how a canister can
-indicate that a relying party is trusted.
-
-## Identify
-
-The window origin (as defined in https://developer.mozilla.org/en-US/docs/Glossary/Origin) is used to identify one
-relying party from another.
-
-> The origin is the concatenation of the protocol and "://", the host name if one exists, and ":" followed by a port
-> number if a port is present and differs from the default port for the given protocol. Examples of typical origins
-> are https://example.org (implying port 443), http://example.net (implying port 80), and http://example.com:8080.
-
-When the signer obtains the origin of the relying party, it **MUST** make sure that it's genuine. Which means that the
-transport channel used between the relying party and signer, must return a origin that can be trusted to be genuine
-e.g. [ICRC-29](./icrc_29_window_post_message_transport.md).
-
-## Verify
-
-When the signer receives a delegation chain request for a global identity, it must verify that the canister targets
-trust the relying party making the request. The signer can use the below method to get a list of trusted origins for
-each canister target and then verify if the relying party is within each list.
-
-### icrc28_get_trusted_origins
+### icrc28_trusted_origins
 
 Returns a list of origins trusted by the canister.
 
 ```
-icrc28_get_trusted_origins : () -> (record { trusted_origins : vec text });
-```
-
-## Supported standards
-
-The signer can use below method to check if the ICRC-28 standard and any related standards are supported.
-
-### icrc28_supported_standards
-
-Returns a list of supported standards related to trusted origins that this canister implements. This query call must not
-require authentication.
-
-```
-icrc28_supported_standards : () -> (vec record { name : text; url : text }) query;
-```
-
-The result should always have at least one entry:
-
-```
-record { name = "ICRC-28"; url = "https://github.com/dfinity/ICRC/blob/main/ICRCs/ICRC-28/ICRC-28.md" }
+icrc28_trusted_origins : () -> (record { trusted_origins : vec text });
 ```
 
 ## Use-Cases
 
-The `icrc28_get_trusted_origins` method is designed to be used with both cold and hot signers.
+There are several reasons why signers might want to get a list of frontend URLs a canister trusts, with one in 
+particular related to removing wallet approval prompts for a user's wallet address.
```

Comment:
I think the removal of wallet prompts is a second order effect. The main goal is to achieve session isolation _with_ a stable principal without signer interactions.

Reply:
I feel like these are the same goals, where one is more clear in the context of web3 than the other. No?

Reply:
I think, we want to be very specific with the concepts we have defined in the standards. The higher-level goal of not prompting the user IMHO is better placed in developer documentation, whereas here we should be very specific to what is actually happening.